samples/fabric/fabric-cli/src/data/event_sub_sample.json

```diff
@@ -8,16 +8,35 @@
             "transactionFunc": "create"
         },
         "event_publication_spec": {
-            "ccArgs": [],
+            "ccArgs": ["keyType", ""],
```

Comment:
What does "keyType" denote?

Reply:
It's just a name for the key imported from remote network.